src/components/HelpDialog.vue

```diff
@@ -99,42 +115,167 @@ export default defineComponent({
       set: (val) => emit("update:modelValue", val),
     });
 
-    const pagedata: Page[] = [
-      {
-        name: "ソフトウェアの利用規約",
-        component: Policy,
-      },
-      {
-        name: "音声ライブラリの利用規約",
-        component: LibraryPolicy,
-      },
-      {
-        name: "使い方",
-        component: HowToUse,
-      },
-      {
-        name: "開発コミュニティ",
-        component: OssCommunityInfo,
-      },
-      {
-        name: "ライセンス情報",
-        component: OssLicense,
-      },
-      {
-        name: "アップデート情報",
-        component: UpdateInfo,
-      },
-      {
-        name: "よくあるご質問",
-        component: QAndA,
-      },
-      {
-        name: "お問い合わせ",
-        component: ContactInfo,
-      },
-    ];
-
-    const selectedPage = ref(pagedata[0].name);
+    // Voicevoxのアップデート確認
```

Comment:
めっちゃ細かいのですが、VOICEVOXというとなにかわからないかもなので、エディタと呼ぶのはどうでしょう。
```suggestion
    // エディタのアップデート確認
```

Reply:
たしかにそうですね。